src/components/accounts/AccountTransactionsScreen.tsx

```diff
@@ -0,0 +1,384 @@
+                <View style={styles.metricContainer}>
+                  <AppText style={styles.metric}>{numFormat(account?.summary?.numUtxos)}</AppText>
+                  <View>
+                    <AppText style={styles.metricLabel}>Spendable{"\n"}Outputs</AppText>
+                  </View>
+                </View>
+                <View style={styles.metricContainer}>
+                  <AppText style={styles.metric}>{numFormat(account?.summary?.satsInMempool)}</AppText>
+                  <View>
+                    <AppText style={styles.metricLabel}>Sats in{"\n"}Mempool</AppText>
+                  </View>
+                </View>
+              </View>
+            </View>
+            <GradientSeparator />
+            <View style={[styles.metrics, styles.metricsNoMargin]}>
+              {/* This view follows the structure of metrics defined above */}
+              {/* its only purpose is to draw underline under transactions */}
+              {/* but place it on top of the GradientSeparator, so it must come after in document order */}
+              <View style={styles.metricContainer}>
+                <View style={styles.metricUnderline}></View>
+              </View>
+              <View style={styles.metricContainer}></View>
+              <View style={styles.metricContainer}></View>
+              <View style={styles.metricContainer}></View>
+            </View>
+          </BackgroundGradient>
+          <View style={styles.transactionsHeaderContainer}>
+            <View style={styles.transactionsHeader}>
+              <TouchableOpacity
+                style={styles.action}
+                activeOpacity={0.7}
+                onPress={onRefresh}
+              >
+                <RefreshIcon width={18} height={18} />                
+              </TouchableOpacity>
+              { refreshing ?
+                <AppText style={[styles.transactionsHeaderText, styles.transactionsHeaderTextRefreshing]}>Updating Parent Account Activity...</AppText> :
+                <AppText style={styles.transactionsHeaderText}>Parent Account Activity</AppText>
+              }
+              <TouchableOpacity
+                style={styles.action}
+                activeOpacity={0.7}
+                onPress={toggleSort}
+              >
+                { sortAsc ?
+                  <UpArrowIcon width={14} height={5} /> :
+                  <DownArrowIcon width={14} height={5} />
+                }
+              </TouchableOpacity>
+            </View>
+          </View>
+          <ScrollView style={styles.transactions}
+            contentContainerStyle={styles.scrollView}
+            refreshControl={
+              <RefreshControl
+                refreshing={refreshing}
+                onRefresh={onRefresh}
+                colors={[Colors.white]}
+                tintColor={Colors.white}
+              />
+            }
+          >
+            { account?.transactions.sort(sortAsc ? txnSortAsc : txnSortDesc).map((txn, i) =>
```

Comment:
I encountered type issue here, `transactions?.sort` would fixed it.

Reply:
Sorry about that, I just refactored transactions from being under account.snapshot.transactions to being just account.transactions, probably why you saw this.  You may need to delete the app/data from your simulator to clear out any accounts you've added with old format since the refactor.

I made the suggested change.  Thanks.